GHE_main.cpp

```diff
@@ -0,0 +1,237 @@
+//
+//  GHE_main.cpp
+//  
+//
+//  Created by Ryan Davies on 6/14/22.
+//
+
+#include <stdio.h>
+#include <string>
+#include <tuple>
+#include <vector>
+#include <numeric>
+#include <cmath>
+using namespace std;
+
+
+/* -------------------SIMPLE MEAN GHE -------------------------------
+Variables (See nomenclature section):
```

Comment:
I would personally just add these as small comments at the point where you create/initialize the variable, that way you don't have to go back to this section to reference it.